openapi/propolis-server.json

```diff
@@ -623,6 +623,14 @@
               "$ref": "#/components/schemas/PciPciBridge"
             }
           },
+          "qemu_pvpanic": {
```

Comment:
So we don't forget later, would you mind updating the `propolis-server-falcon.json` version of the spec as well? 

FYI: The build with the `falcon` feature must be done on illumos for FFI/linking reasons today, unlike "vanilla" which succeeds on linux with stubs.